Module2-AspNetCore/README.md

```diff
@@ -764,7 +775,9 @@ In this task, you'll learn how the ASP.NET Core project templates use ASP.NET Id
 
 	_Account register view_
 
-1. After creating a new user and applying the database migrations, stop the application and explore the database by navigating to the **(localdb)MSSQLLocalDB/Databases/aspnet5-MyWebApp-<guid>/Tables** in the **SQL Server Object Explorer** view. Right-click the **dbo.AspNetUsers** table and select **View Data** to see the properties of the user you created.
+1. After you register your first user, you'll notice an error message suggesting that you apply existing migrations. Click **Apply Migrations**. You will now see that you're logged in as the new user.
```

Comment:
I was surprised by the database error page. It's better to be more verbose here so customers aren't stressed out.